spec/lib/chat_channel_fetcher_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
```suggestion
```

Minor note, this is no longer required going forward per https://github.com/discourse/discourse/blob/3d62e335f7590f5f96e31b2903e8dde2f991e51e/.rspec#L2